lib/listener-tracker.js

```diff
@@ -175,9 +206,14 @@ SubscriptionTracker.prototype = {
         }
 
         if (!wrapper) {
-            wrapper = new EventEmitterWrapper(target);
+            if (isNonEventEmitter(target)) {
+              nonEE = new EventEmitterAdapter(target);
+              destroyEvent = DOM_NODE_REMVOED_FROM_DOCUMENT;
```

Comment:
Yeah, that's true. I was actually struggling with that because even `DOMNodeRemovedFromDocument` is deprecated and the replacement is to use [mutation observers](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver), which doesn't look simple to implement within this solution.

So I would be good not setting the listener here.